lang/es-es.js

```diff
@@ -22,32 +22,32 @@ export default {
 	"components.input-date-time-range.endDate": "Fecha final",
 	"components.input-date-time-range.errorBadInput": "{startLabel} debe ser anterior a {endLabel}",
 	"components.input-date-time-range.startDate": "Fecha de inicio",
-	"components.input-date-time.date": "",
-	"components.input-date-time.errorMaxDateOnly": "La fecha debe ser anterior al {maxDate}",
-	"components.input-date-time.errorMinDateOnly": "La fecha debe ser posterior al {minDate}",
-	"components.input-date-time.errorOutsideRange": "La fecha debe estar entre el {minDate} y el {maxDate}",
-	"components.input-date-time.time": "",
-	"components.input-date.clear": "",
+	"components.input-date-time.date": "Fecha de la última actividad",
+	"components.input-date-time.errorMaxDateOnly": "La fecha debe ser anterior a {maxDate}",
+	"components.input-date-time.errorMinDateOnly": "La fecha debe ser posterior a {minDate}",
+	"components.input-date-time.errorOutsideRange": "La fecha debe estar entre {minDate} y {maxDate}",
```

Comment:
Before: `The date must be between {minDate} and {maxDate}`
After: `The date must be between {minDate} and {maxDate}`
Seems fine